WalletWasabi.Fluent/ViewModels/SettingsPageViewModel.cs

```diff
@@ -1,15 +1,28 @@
 using ReactiveUI;
 using System.Windows.Input;
+using WalletWasabi.Gui.Validation;
+using WalletWasabi.Models;
 
 namespace WalletWasabi.Fluent.ViewModels
 {
 	public class SettingsPageViewModel : NavBarItemViewModel
 	{
+		private string _randomString;
+
 		public SettingsPageViewModel(IScreen screen) : base(screen)
 		{
 			Title = "Settings";
 
 			NextCommand = ReactiveCommand.Create(() => screen.Router.Navigate.Execute(new HomePageViewModel(screen)));
+
+			this.ValidateProperty(x => x.RandomString, (IValidationErrors errors) => errors.Add(ErrorSeverity.Error, "Random Error Message"));
```

Comment:
> error CS0311: The type 'WalletWasabi.Fluent.ViewModels.SettingsPageViewModel' cannot be used as type parameter 'TSender' in the generic type or method 'ValidationExtensions.ValidateProperty<TSender, TRet>(TSender, Expression<Func<TSender, TRet>>, ValidateMethod)'. There is no implicit reference conversion from 'WalletWasabi.Fluent.ViewModels.SettingsPageViewModel' to 'WalletWasabi.Gui.Validation.IRegisterValidationMethod'.

Reply:
#4535 needs to be merged. I just worked ahead here.